Migration OCP 3 to OCP 4 (#108)
mramendi/kie-docs: doc-content/enterprise-only/openshift/migration-ocp3-4-bc-proc.adoc

[id='migration-ocp3-4-bc-proc-{context}']
= Migrating information in {CENTRAL}

If you have an existing authoring environment in {OPENSHIFT} version 3, you can copy the `.niogit` repository and the Maven repository from {CENTRAL} in this environment to {CENTRAL} in a new deployment on {OPENSHIFT} version 4. This action makes all the same projects and artifacts available in the new deployment.

.Prerequisites

* You must have a machine that has network access to both the {OPENSHIFT} version 3 and {OPENSHIFT} version 4 infrastructures.

> * The `oc` command-line client from {OPENSHIFT} version 4 must be installed on the machine.

Reviewer: Maybe link to the instructions to get the `oc` command? Just a suggestion

.Procedure

. Ensure that no web clients and no client applications are connected to any elements of the old and new deployment, including {CENTRAL} and {KIE_SERVERS}.

> . Create a temporary directory and change into this directory.

Reviewer 2: *this* is confusing.

```suggestion
. Create a temporary directory and change into the new directory.
```

. Using the `oc` command, log in to the {OPENSHIFT} version 3 infrastructure and switch to the project containing the old deployment.

> . To view the pod names in the old deployment, run the command:

Reviewer 2: ```suggestion
. To view the pod names in the old deployment, run the following command:
```

+
----
oc get pods
----
+
Find the {CENTRAL} pod. The name of this pod includes `{PRODUCT_INIT}centr`. In a high-availability deployment, you can use any of the {CENTRAL} pods.
+
. Use the `oc` command to copy the the `.niogit` repository and the Maven repository from the pod to the local machine, for example:
+
[subs="attributes,verbatim,macros,specialchars,replacements"]
----
oc cp myapp-{PRODUCT_INIT}centr-5-689mw:/opt/kie/data/.niogit .niogit
oc cp myapp-{PRODUCT_INIT}centr-5-689mw:/opt/kie/data/maven-repository maven-repository
----
+
. Using the `oc` command, log in to the {OPENSHIFT} version 4 infrastructure and switch to the project containing the new deployment.

> . To view the pod names in the new deployment, run the command:

Reviewer 2: ```suggestion
. To view the pod names in the new deployment, run the following command:
```
